todo/src/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import registerServiceWorker from './registerServiceWorker';
+
+
+
+
+const store = createStore(reducer);
+
+
+ReactDOM.render(
+    <Provider store={store}>
```

Comment:
Great way to wrap app in a Redux Provider. This is essentially what gives the whole app and its nested components acess to the Redux Store created above.